core/controlplane/config/templates/cloud-config-worker

```diff
@@ -416,14 +428,15 @@ ssh_authorized_keys:
       content: |
         [Unit]
         Description=Pull and tag a mirror image for pause-amd64
-        After=docker.service
```

Comment:
What is the intention behind the removal of this line?
You probably want to ensure that `docker.service` is up and running before running `pause-amd64.service`?
Then, `After=docker.service` + `Wants=docker.service` + `retry bash -c '/usr/bin/systemctl is-active docker.service && /usr/bin/docker pull ...'` would be what you want/my recommendation.
If you don't care about temporary `docker.service` failures, i.e. no retrying wanted, `After=docker.service` + `Requires=docker.service` + `/usr/bin/docker pull ...` would be sufficient. However, can we really assume `docker.service` to start reliably?